aspnetcore/security/cors.md

```diff
@@ -59,8 +59,7 @@ You can specify a cross-origin policy when adding the CORS middleware using the
 
 [!code-csharp[Main](cors/sample/CorsExample1/Startup.cs?highlight=11,12&range=22-38)]
 
-> [!NOTE]
-> URL must be specified without trailing slash otherwise comparison will return false and no header will be returned.
+**Note:** The URL must be specified without trailing slash. If the URL terminates with `/`, the comparison will return `false` and no header will be returned.
```

Comment:
Shouldn't it be: without "a" trailing slash?

Reply:
I didn't notice your second commit!

Reply:
Present tense version:

> The URL is specified without trailing slash. If the URL terminates with `/`, the comparison returns `false` and no header is returned.